src/redux/store.js

```diff
@@ -0,0 +1,14 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { configureStore } from '@reduxjs/toolkit';
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
- [ ] Please remove any eslint disablement lines from your code and fix its errors yourself.
[Hint] If you remove the curly braces around `configureStore` the problem will be fixed.